<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spectrumCalculatorDialog</class>
 <widget class="QDialog" name="spectrumCalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recalculate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="formulaXLabel">
         <property name="text">
          <string>Formula for x values:</string>
         </property>
         <property name="buddy">
          <cstring>formulaX</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="formulaX"/>
       </item>
       <item>
        <widget class="QLabel" name="errorsX">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="formulaYLabel">
         <property name="text">
          <string>Formula for y values:</string>
         </property>
         <property name="buddy">
          <cstring>formulaY</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="formulaY"/>
       </item>
       <item>
        <widget class="QLabel" name="errorsY">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="toNm">
         <property name="text">
          <string>nm/cm-1 conversion</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="toAbsorption">
         <property name="text">
          <string>To absorption</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toTransmittance">
         <property name="text">
          <string>To transmittance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>formulaX</tabstop>
  <tabstop>toNm</tabstop>
  <tabstop>formulaY</tabstop>
  <tabstop>toAbsorption</tabstop>
  <tabstop>toTransmittance</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>spectrumCalculatorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>spectrumCalculatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
